docs/03-tutorials/00-serverless/svls-09-log-into-private-packages-registry.md

```diff
@@ -126,7 +126,7 @@ You should get a result similar to the this example:
 
 ```bash
 NAME            CONFIGURED   BUILT     RUNNING   RUNTIME    VERSION   AGE
-test-function   True         True      True      nodejs12   1         96s
+test-function   True         True      True      nodejs14s   1         96s
```

Comment:
```suggestion
test-function   True         True      True      nodejs14   1         96s
```
Is the `s` at the end of `nodejs14` required?

Reply:
No, that was typo, thank you for catching it!